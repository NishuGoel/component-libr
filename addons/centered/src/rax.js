/** @jsx createElement */
// eslint-disable-next-line import/no-extraneous-dependencies
import { createElement } from 'rax';
// eslint-disable-next-line import/no-extraneous-dependencies
import View from 'rax-view';
import styles from './styles';

export default function(storyFn) {
  return (
    <View style={styles.style}>
      <View style={styles.innerStyle}>{storyFn()}</View>
    </View>
  );
}

if (module && module.hot && module.hot.decline) {
  module.hot.decline();
}
