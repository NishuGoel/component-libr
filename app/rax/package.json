{
  "name": "@storybook/rax",
  "version": "5.2.0-alpha.44",
  "description": "Storybook for Rax: Develop Rax Component in isolation.",
  "keywords": [
    "storybook",
    "rax"
  ],
  "homepage": "https://github.com/storybookjs/storybook/tree/master/app/rax",
  "bugs": {
    "url": "https://github.com/storybookjs/storybook/issues"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/storybookjs/storybook.git",
    "directory": "app/rax"
  },
  "license": "MIT",
  "main": "dist/client/index.js",
  "jsnext:main": "src/client/index.js",
  "bin": {
    "build-storybook": "./bin/build.js",
    "start-storybook": "./bin/index.js",
    "storybook-server": "./bin/index.js"
  },
  "scripts": {
    "prepare": "node ../../scripts/prepare.js"
  },
  "dependencies": {
    "@storybook/core": "5.2.0-alpha.44",
    "babel-preset-rax": "^1.0.0-beta.0",
    "common-tags": "^1.8.0",
    "core-js": "^2.6.2",
    "global": "^4.3.2",
    "regenerator-runtime": "^0.12.1"
  },
  "devDependencies": {
    "rax": "^0.6.5"
  },
  "peerDependencies": {
    "babel-loader": "^7.0.0 || ^8.0.0",
    "rax": "^0.4.0 || ^1.0.0"
  },
  "publishConfig": {
    "access": "public"
  }
}
