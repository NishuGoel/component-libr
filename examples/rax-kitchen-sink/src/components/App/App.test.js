import { createElement } from 'rax';
import renderer from 'rax-test-renderer';
import App from './index';

test('App render correctly', () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});
