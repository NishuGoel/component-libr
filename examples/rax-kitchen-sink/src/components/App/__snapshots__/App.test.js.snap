// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render correctly 1`] = `
Object {
  "children": Array [
    Object {
      "children": Array [
        Object {
          "children": Array [
            "Welcome to Rax",
          ],
          "style": Object {
            "alignContent": "flex-start",
            "border": "0 solid black",
            "boxSizing": "border-box",
            "display": "block",
            "flexDirection": "column",
            "flexShrink": 0,
            "fontSize": 32,
            "position": "relative",
            "whiteSpace": "pre-wrap",
          },
          "tagName": "SPAN",
        },
      ],
      "style": Object {
        "alignContent": "flex-start",
        "border": "0 solid black",
        "boxSizing": "border-box",
        "display": "flex",
        "flexDirection": "column",
        "flexShrink": 0,
        "position": "relative",
      },
      "tagName": "DIV",
    },
    Object {
      "children": Array [
        "To get started, edit src/components/App.js and save to reload.",
      ],
      "style": Object {
        "alignContent": "flex-start",
        "border": "0 solid black",
        "boxSizing": "border-box",
        "display": "block",
        "flexDirection": "column",
        "flexShrink": 0,
        "fontSize": 32,
        "position": "relative",
        "whiteSpace": "pre-wrap",
      },
      "tagName": "SPAN",
    },
  ],
  "style": Object {
    "alignContent": "flex-start",
    "border": "0 solid black",
    "boxSizing": "border-box",
    "display": "flex",
    "flexDirection": "column",
    "flexShrink": 0,
    "position": "relative",
  },
  "tagName": "DIV",
}
`;
